scipy/cluster/vq.py

```diff
@@ -563,15 +582,25 @@ def _kpp(data, k):
     dims = data.shape[1] if len(data.shape) > 1 else 1
     init = np.ndarray((k, dims))
 
+    # get random generator API
```

Comment:
We actually have this in `_util:rng_integers`. I also discovered that recently.

Reply:
Thanks. I used it. and I also added `_util:rng_random` and a test for it. PTAL again.